linera-sdk/src/lib.rs

```diff
@@ -114,6 +114,16 @@ pub trait Contract: WithContractAbi + ContractAbi + Send + Sized {
     /// from another chain and accepted in a block.
     type Message: Serialize + DeserializeOwned + Send + Sync + Debug + 'static;
 
+    /// Immutable parameters specific to this application (e.g. the name of a token).
```

Comment:
Not sure if the fungible token example is helpful here: A user who is thinking of a completely different application might find this confusing.